containers/daap-landing-to-raw/src/var/task/landing_to_raw.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import boto3
+from data_platform_logging import DataPlatformLogger, s3_security_opts
+from data_platform_paths import get_raw_data_bucket
+
+logger = DataPlatformLogger(
+    extra={
```

Comment:
do we still need this `extra` argument? I thought we'd moved this into the logging module

Reply:
I removed it, thanks

Reply:
Do still need the extra args i think, the `security_opts` were removed

Reply:
Yeah it doesn't set the base image or lambda container image versions within the logging module. It does get name of the lambda from an environment variable but also if you wanted to add in data_product_name and table_name you would do it in extra too

Reply:
ok reverted